src/darwin/Framework/CHIPTests/MTRDeviceTests.m

```diff
@@ -1438,6 +1438,10 @@ - (void)test017_TestMTRDeviceBasics
 
     // Check that device resets start time on subscription drop
     XCTAssertNil(device.estimatedStartTime);
+
+    // Issue a read and expect no actual read from device - verified in logs during run
```

Comment:
We should be able to do better here:

1) Set up a report handler that will XCTFail if it's called.
2) Create an expectation.
3) Start a timer to fulfill the expectation after 3s or so.
4) Wait for the expectation.

Then if we get a read report during that 3s we will fail the test, which I think is what we want.